core-curriculum/knowledge/scala.json

```diff
@@ -50,13 +87,31 @@
           "type": "article",
           "url": "https://www.scala-lang.org/files/archive/spec/2.11/08-pattern-matching.html",
           "tags": ["official"]
+        },
+        {
+          "name": "Pattern matching",
+          "type": "video",
+          "url": "https://www.coursera.org/learn/progfun1/lecture/cdHAM/lecture-4-6-pattern-matching",
+          "tags": ["coursera", "mooc"]
         }
+      ],
+
+      "abilities": [
+        "Decompose code by matching in a more descriptive way",
+        "Recognize when it better to use Pattern matching and when to use if/else",
```

Comment:
Suggested Rewording: Recognise when to use Pattern Matching over If/else expressions.